ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/ethernet_interfaces.py

```diff
@@ -139,8 +139,23 @@ def _get_ethernet_interface_cfg(self, adapter: dict, node_index: int, connected_
                     },
                 }
             )
-            if get(adapter, "port_channel.lacp_fallback.mode") == "static":
+            if get(adapter, "port_channel.lacp_fallback.mode") == "static" or get(adapter, "port_channel.lacp_fallback.mode") == "individual":
                 ethernet_interface["lacp_port_priority"] = 8192 if node_index == 0 else 32768
+                if get(adapter, "port_channel.lacp_fallback.mode") == "individual" and get(adapter, "port_channel.lacp_fallback.individual") is not None:
+                    # check if the referred name exists in the global evpn_vlan_bundles
+                    if (profile := get_item(self._hostvars["port_profiles"], "profile", get(adapter, "port_channel.lacp_fallback.individual.profile"))) is None:
+                        raise AristaAvdMissingVariableError(
+                            "The 'profile' of every port-channel lacp fallback individual setting must be defined in the 'port_profiles'. First occurence seen"
+                            f" of a missing profile is '{get(adapter, 'port_channel.lacp_fallback.individual.profile')}' for the connected endpoint with the"
+                            f" name '{connected_endpoint['name']}'."
+                        )
+
+                    ethernet_interface["type"] = "switched"
+                    ethernet_interface["mode"] = profile.get("mode")
+                    ethernet_interface["vlans"] = profile.get("vlans")
+                    ethernet_interface["native_vlan_tag"] = profile.get("native_vlan_tag")
+                    ethernet_interface["native_vlan"] = profile.get("native_vlan")
```

Comment:
I think we may have to split out the config from the "NOT a port-channel member" section below so we can have call a common function for individual or non-member ports. (The function should take the adapter for non-member ports and the profile for indivual.

Reply:
created a shared function which can be used by both